ReactNativeTemplate/android/app/build.gradle

```diff
@@ -109,6 +109,11 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
+            defaultConfig {
+                targetSdkVersion 22
```

Comment:
If there's a fix in `SalesforceReactActivity` that lets us keep `targetApiVersion = 25`, I would prefer that.